cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -12,7 +12,7 @@ DESCRIPTION
 
 OPTIONS
        --input <proto-file-path>
```

Comment:
```suggestion
       --input  <path>
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/fdbf3d28db757767a7ef880cc166507ea51caaa7